media/robots.txt

```diff
@@ -6,5 +6,30 @@ Request-rate: 1/5
 Disallow: /*feed=rss
 Disallow: /*type=feed
 Disallow: /skins
-Disallow: /template:
+Disallow: /*docs/feeds
+Disallow: /*docs/templates
+Disallow: /*docs*Template
```

Comment:
will this block https://developer.mozilla.org/en-US/docs/Mozilla/Tech/XUL/Tutorial/Templates?